.github/renovate.json

```diff
@@ -30,6 +30,10 @@
       "groupName": "all dependencies",
       "groupSlug": "all"
     },
+    {
+      "matchPackageNames": ["@percy/puppeteer"],
+      "allowedVersions": "<=2.0.0"
+    },
```

Comment:
See related issue for Jest/ESM support, which was added to `@percy/puppeteer` on `v2.0.1`.

https://github.com/percy/percy-puppeteer/issues/426

For now, we downgrade to `v2.0.0`.